dags/atd_kits_sig_stat_pub.py

```diff
@@ -17,7 +17,7 @@
     "email_on_failure": False,
     "email_on_retry": False,
     "retries": 0,
-    "retry_delay": duration(minutes=5),
+    "execution_timeout": duration(minutes=5),
```

Comment:
i have removed the pointless `retry_delay` wherever it's used with zero `retries`.